examples_gbs/run_tutorial_points.py

```diff
@@ -6,25 +6,26 @@
 This tutorial shows how to generate GBS point process samples and use them to detect outlier
 points in a data set. Point processes are models for generating random point patterns and GBS
 devices can be programmed to operate as special types of point processes that generate clustered
-random point patterns.
+random point patterns :cite:`jahangiri2019point`.
 
-In GBS point processes, the probability of generating a specific pattern of points depends on
+The probability of generating a specific pattern of points in GBS point processes depends on
 matrix functions of a kernel matrix that describes the similarity between the points. Matrix
-functions that appear in GBS point processes are typically permanents and hafnians. In this
-tutorial, we use the permanental point process in which the probability of observing a pattern of
-points :math:`S` depends on the permanent of their corresponding kernel matrix :math:`K_S` as:
+functions that appear in GBS point processes are typically
+`permanents <https://en.wikipedia.org/wiki/Permanent_(mathematics)>`__ and
+`hafnians <https://en.wikipedia.org/wiki/Hafnian>`__. In this tutorial, we use the permanental point
```

Comment:
It's better to link to the walrus for information about the hafnian https://hafnian.readthedocs.io/en/latest/hafnian.html